IOPool/Common/bin/EdmProvDump.cc

```diff
@@ -1055,21 +1055,76 @@ int main(int argc, char* argv[]) {
+                                                                                   ")")(kDontPrintProductsCommandOpt,
+                                                                                        "do not "
+                                                                                        "print "
+                                                                                        "products "
+                                                                                        "produced "
+                                                                                        "by "
+                                                                                        "modul"
+                                                                                        "e")(kDumpPSetIDCommandOpt,
+                                                                                             value<std::string>(),
+                                                                                             "print "
+                                                                                             "the "
+                                                                                             "parame"
+                                                                                             "ter "
+                                                                                             "set "
+                                                                                             "associ"
+                                                                                             "ated "
+                                                                                             "with "
+                                                                                             "the "
+                                                                                             "parame"
+                                                                                             "ter "
+                                                                                             "set "
+                                                                                             "ID "
+                                                                                             "string"
+                                                                                             " (and "
+                                                                                             "print "
+                                                                                             "nothin"
+                                                                                             "g "
+                                                                                             "else"
+                                                                                             ")");
```

Comment:
This looks worse than it was before. Several other of the modifications look worse, although this is the worst one  of all of them.

Reply:
wow, auto formating :-) yes this look bad

Reply:
and few other places too :-(